Jenkinsfile

```diff
@@ -0,0 +1,80 @@
+#!groovy
+
+// https://github.com/feedhenry/fh-pipeline-library
+@Library('fh-pipeline-library') _
+
+def repositoryName = "unifiedpush-apb"
+def projectName = "test-${repositoryName}-${currentBuild.number}-${currentBuild.startTimeInMillis}"
+
+stage('Trust') {
+    enforceTrustedApproval('aerogear')
+}
+
+node ("ocp-slave") { 
+    stage('Cleanup') {
+        deleteDir()
+    }
+    stage('Cloning the repo') {
+        checkout scm
+    }
+
+    try {
+    
+        stage('Create OpenShift project') {
+            sh script: "oc new-project ${projectName}"
+        }
+        stage('Build APB') {
+            try {
+                timeout(5) {
+                    sh script: """
+                    oc new-build --name ${repositoryName} --binary
+                    oc start-build ${repositoryName} --from-dir=.
+                    sleep 10
+                    oc logs -f bc/${repositoryName}
+                    """
+                }
+            } catch (Exception e) {
+                error "Build timed out"
+            }
+
+        }
+
+        stage('Add admin policy to serviceaccount "default"') {
+            sh script: "oc policy add-role-to-user admin --serviceaccount=default"
```

Comment:
Can't we have this on the slave already preset? This seems to be cluster-wide and not just per/project.

Reply:
any comments, @psturc

Reply:
So this command adds the admin role to the `default` service account only for the scope of the project. It is not cluster-wide.